src/styles/card-container.scss

```diff
@@ -0,0 +1,144 @@
+.card-container {
+  color: #060b35;
+  gap: 48px 46px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+
+  place-content: center;
+
+  padding: 50px 40px;
+}
+
+@media (min-width: 488px) {
+  .card-container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .card-container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .card-container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.product-card {
+  display: flex;
+  box-sizing: border-box;
+  width: 200px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  transition: all 0.3s;
+}
+
+.product-card:hover {
+  transform: scale(1.2);
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+  align-self: center;
+  margin-inline: 3px;
+}
+
+.product-card__title {
+  text-align: left;
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.product-card:hover .product-card__title {
+  color: #34568b;
+}
```

Comment:
The hover effect changes the color of the card title, but it's not clear why. This could confuse users, as they might expect the title to be clickable. If it's not intended to be clickable, consider removing this effect.